src/common/iop_order.c

```diff
@@ -34,6 +34,8 @@
 #define DT_IOP_ORDER_VERSION 4
 
 #define DT_IOP_ORDER_INFO FALSE	// used while debugging
+#define DT_ONTHEFLY_INFO FALSE   // while debugging on-the-fly conversion
+#define DT_ONTHEFLY_WRITING FALSE // If TRUE will do history update
```

Comment:
And of course we need now to activate it. No?

Reply:
done